### DISCUSSION TITLE
User unicity when creating a one time account

### DISCUSSION THREAD

**Discussion by wmarie-joseph** [oliverklee/ext-onetimeaccount#644] [General]
Hello @oliverklee ,

I'm using onetimeaccount to enable users to create an account before registering to an event (seminars extension).

Users can create an unlimited amount of accounts with the same email/username, is there an option or a way to prevent a user from creating an account if a fe_user with the same email exists ?

Thanks in advance.

Best regards,

W

**Comment by oliverklee** [comment]
Hi @wmarie-joseph, no, this currently is not possible.

Such a feature would also be problematic for two reasons:
1. It would be possible to find out which email addresses have already been used for a registration (which probably would be a privacy violation and possibly a security risk).
2. It would prevent users from registering for a second time (because they cannot log in to the FE user they created via onetimeaccount when they registered for their first event).

If you want users who register for multiple events to use the same FE user, the recommended way would be to use a regular FE login. (If you like, you can also combine both on the login page: the regular FE login form and onetimeaccount.)

There probably is a way for your preferred self-registration extension to use the email address as user name and to prevent double registrations for the same email address.

**Reply by wmarie-joseph** [reply]
Ok i see, thank you.
